### DISCUSSION TITLE
libglib2.0-0 (>= 2.39.4)

### DISCUSSION THREAD

**Discussion by Nickinton** [linuxmint/discussions#556] [Questions]
When trying to install Minecraft Launcher's .deb package, a message appears saying this file (in the title) is missing, and I can't satisfy the dependency.
![Captura de pantalla de 2024-07-30 17-26-35](https://github.com/user-attachments/assets/e61e6b23-8ed4-4fe9-8454-b855680df380)

**Comment by rcalixte** [comment]
Including versions, what operating systems does this Minecraft Launcher support? What version and what operating system are you running? Chances are that the operating system you are using for this installer are unsupported.

Assuming you are on a freshly updated system (e.g. Mint 22), you will need the launcher to be updated to support the new base of Ubuntu 24.04.

**Reply by Nickinton** [reply]
- **what operating systems does this Minecraft Launcher support?**: It doesn't say (or I can't find it), just says: Debian (or Debain based distr.)
- **What version and what operating system are you running?**: Linux Mint 22 ([System info](https://termbin.com/j7n6))
- **Assuming you are on a freshly updated system (e.g. Mint 22), you will need the launcher to be updated to support the new base of Ubuntu 24.04**: it's like you read my mind (...or my PC in this case). So I should wait until they update the launcher's compatibility?

**Reply by rcalixte** [reply]
> * It doesn't say (or I can't find it), just says: Debian (or Debain based distr.)

That unfortunately isn't helpful. If it is Debian 12, that is a year old at this point. Debian 13 likely will not be released until next year with similar changes that made it into Ubuntu 24.04/Mint 22.



> * So I should wait until they update the launcher's compatibility?

I presume it's not open source. (I'm only marginally familiar with the game.) If it is not open source, your only option is to wait.

If the game is important enough to you and you have a backup from before the upgrade to Mint 22, you can rollback until this update is ready.

**Reply by Nickinton** [reply]
Thanks.
